scripts/lib/CIME/case/check_input_data.py

```diff
@@ -83,6 +83,21 @@ def stage_refcase(self, input_data_root=None, data_list_dir=None):
         rundir       = self.get_value("RUNDIR")
 
         refdir = os.path.join(din_loc_root, run_refdir, run_refcase, run_refdate)
+        expect(os.path.isdir(refdir),
```

Comment:
This comment can be removed, the ref data is now automatically downloaded if it exists on the server.